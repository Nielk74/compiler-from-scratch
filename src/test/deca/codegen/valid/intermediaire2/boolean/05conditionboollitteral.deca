//Exécute le if
{
if((true && (4 == 5)) || false || (true == true)){
  println("(true && (4 == 5)) || false || (true == true)");
}else{
  println("!((true && (4 == 5)) || false || (true == true))");
}

//Exécute le else

if((true && (4 == 5)) || false || (true == false)){
  println("(true && (4 == 5)) || false || (true == false)");
}else{
  println("!((true && (4 == 5)) || false || (true == false))");
}
}
