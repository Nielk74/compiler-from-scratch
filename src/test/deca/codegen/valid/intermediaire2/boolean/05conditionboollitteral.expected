(true && (4 == 5)) || false || (true == true)
!((true && (4 == 5)) || false || (true == false))
